src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -956,6 +957,116 @@ TEST_F(MessageHelperTest, SubscribeApplicationToSoftButton_CallFromApp) {
       message_params, appSharedPtr, function_id);
 }
 
+TEST(MessageHelperTestCreate, CreateHashUpdateNotification_FunctionId_Equal) {
+  const uint32_t kApplicationId = 0;
+  const smart_objects::SmartObjectSPtr kNotificationSPtr =
+      MessageHelper::CreateHashUpdateNotification(kApplicationId);
+
+  ASSERT_TRUE(kNotificationSPtr.valid());
+
+  const smart_objects::SmartObject& kNotificationRef = *kNotificationSPtr;
+
+  const int kFunctionId =
+      static_cast<int>(mobile_apis::FunctionID::OnHashChangeID);
+  const int kMessageType =
+      static_cast<int>(::application_manager::kNotification);
+
+  EXPECT_EQ(kFunctionId,
+            kNotificationRef[strings::params][strings::function_id].asInt());
+  EXPECT_EQ(
+      kApplicationId,
+      kNotificationRef[strings::params][strings::connection_key].asUInt());
+  EXPECT_EQ(kMessageType,
+            kNotificationRef[strings::params][strings::message_type].asInt());
+}
+
+TEST(MessageHelperTestCreate, CreateAppVrHelp_AppName_Equal) {
+  const MockApplicationSharedPtr kMockApplicationSPtr =
+      utils::MakeShared<MockApplication>();
+
+  application_manager::CommandsMap commands_map;
+  DataAccessor< ::application_manager::CommandsMap> data_accessor(commands_map,
+                                                                  true);
+
+  const smart_objects::SmartObject* kVrSynonymsPrt = NULL;
+  const utils::custom_string::CustomString kApplicationName("213");
+
+  EXPECT_CALL(*kMockApplicationSPtr, name())
+      .WillOnce(ReturnRef(kApplicationName));
+  EXPECT_CALL(*kMockApplicationSPtr, vr_synonyms())
+      .Times(AtLeast(1))
+      .WillRepeatedly(Return(kVrSynonymsPrt));
+  EXPECT_CALL(*kMockApplicationSPtr, commands_map())
+      .WillOnce(Return(data_accessor));
+
+  const smart_objects::SmartObjectSPtr kNotificationPtr =
+      MessageHelper::CreateAppVrHelp(kMockApplicationSPtr);
+
+  ASSERT_TRUE(kNotificationPtr.valid());
+
+  EXPECT_EQ(kApplicationName,
+            (*kNotificationPtr)[strings::vr_help_title].asString());
+}
+
+TEST_F(MessageHelperTest,
+       MobileResultFromString_StringValueOfEnum_CorrectEType) {
+  MobileResults::eType tested_enum;
+  MobileResults::eType converted;
```

Comment:
@Kozoriz should it been named **converted_enum** like similar variable tested_enum?